docs (fix): fixed weird line breaks in zeebe health docs (#8323)
camunda/camunda-docs: docs/components/zeebe/technical-concepts/health.md

## Unhealthy

When a node in a Zeebe cluster is unhealthy, it means parts of the system may not work. This is often a transient state: for example, when a node is starting, it starts in an unhealthy state, as several components (e.g. a partition, the workflow engine, etc.) are not yet installed or ready for operation.

When this is temporary, and the cluster overall converges to a steady healthy state, you can safely ignore these temporary signals. However, if things persist, or if you see flip-flopping between healthy and unhealthy, it can be an indicator of an underlying issue which requires investigation.

:::note
Note that even if parts of the cluster are unhealthy, it can be that other parts are still working fine.
:::

## Dead

> When something is marked as dead in a cluster, it means it failed in a non-recoverable way. This means it **will** require human intervention to recover. This is a rare status, but it can happen if data corruption is detected, for example. We recommend prompt investigation into any components with a dead status.

Reviewer: ```suggestion
When something is marked as dead in a cluster, it means it failed in a non-recoverable way. This means it **will** require human intervention to recover. This is a rare status, but it can happen if data corruption is detected, for example. You should promptly investigate any components with a dead status.
```
